graphql_compiler/compiler/compiler_frontend.py

```diff
@@ -398,7 +404,8 @@ def _compile_vertex_ast(schema, current_schema_type, ast,
             # Make sure there's a tag right before it for the optional Backtrack to jump back to.
```

Comment:
let's make `tag` -> `marked location` in this entire comment to avoid confusion with `@tag` directives